src/cli/node/with_node.rs

```diff
@@ -255,7 +255,7 @@ pub enum Command {
         /// Socket address of the Witnet node to query.
         #[structopt(short = "n", long = "node")]
         node: Option<SocketAddr>,
-        /// Show all the information about utxos
+        /// Show all the information about utxos.
```

Comment:
I thought we wanted to remove dots?

Reply:
I thought dots in comments were fine since those messages are not going to be printed

Reply:
These comments are used to document the flags of each subcommand, if you run `witnet node getUtxoInfo --help` you will see a detailed description of each. So yes, they are printed, but I'm not sure if they should end with a dot or not.

Reply:
ahhh ok, no dots then :)

Reply:
As an example, `git --help` does not have dots, but `git add --help` has dots when explaining the flags.